### PR TITLE
Type, Primary key and inconsistency fixes

### PR DESCRIPTION
**Pull request by kvij**
The second commit on findId is not that important to me so If you think the regex is overkill leave that out. I do use findId to resolve Division by HID and Exact does not like it when it is forced to be a string.

I am working on lazy loading the deferred but need consistent classnames for that (see lazyload branch). As most inconsistencies where posted recently (in entities that I use even) I wanted to push this out as soon as possible.

Please let me know if there is anything you disagree with.

### CODE REVIEW

File: src/Picqer/Financials/Exact/Division.php [hunk 1]
```diff
@@ -32,6 +32,8 @@ class Division extends Model
 {
 
     use Query\Findable;
+    
+	protected $primaryKey = 'Code';
```
Comment (stephangroen) [inline src/Picqer/Financials/Exact/Division.php]: Can you fix this indentation?

File: src/Picqer/Financials/Exact/ReceivableList.php [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+ * @property String $JournalCode Code of Journal
+ * @property String $JournalDescription Description of Journal
+ * @property String $YourRef Your reference
+ */
 class ReceivableList extends Model
 {
 
     use Query\Findable;
 
+	protected $primaryKey = 'HID';
```
Comment (stephangroen) [inline src/Picqer/Financials/Exact/ReceivableList.php]: Can you fix this indentation?

File: src/Picqer/Financials/Exact/Transactions.php [hunk 3]
```diff
@@ -1,54 +1,6 @@
...
-        'Status',
-        'StatusDescription',
-        'TransactionLines',
-        'Type',
-        'TypeDescription'
-    ];
-    protected $url = 'financialtransaction/Transactions';
-}
\ No newline at end of file
+class_alias(__NAMESPACE__ . '\Transaction', __NAMESPACE__ . '\Transactions');
```
Comment (stephangroen) [inline src/Picqer/Financials/Exact/Transactions.php]: You have created an extra class for backward compatibility here but have not done so for the Subscription* classes.

File: src/Picqer/Financials/Exact/Division.php (hunk 1 above)
Comment (a-okay) [inline src/Picqer/Financials/Exact/Division.php]: Sorry just updated Komodo Edit will check its settings.

File: src/Picqer/Financials/Exact/Transactions.php (hunk 3 above)
Comment (a-okay) [inline src/Picqer/Financials/Exact/Transactions.php]: As Subscriptions and SubscriptionTypes and SubscriptionLines are added recently added I dit not think this nessesary. Do you want me to add aliases for these classes as well?

Comment (stephangroen) [inline src/Picqer/Financials/Exact/Transactions.php]: Since they are already released, we should. If you can do that, it would be great :)

Comment (a-okay) [inline src/Picqer/Financials/Exact/Transactions.php]: I have just added them. All issues with this pull request should be resolved unless I introduced something in a5da440 or more likely 87a2e70 that you do not approve off.

Comment (a-okay) [inline src/Picqer/Financials/Exact/Transactions.php]: I don't see a use case that does not use the Autoloader so I could add a trigger_error statement with a depreciation warning. This way we can in the future clean up the extra files.

**Comment by stephangroen** [comment]
Thanks @kvij :)

**Pull request merged**